### PR TITLE
refactor: slight optimisation

### PR DESCRIPTION
**Pull request by MoskalykA**

### CODE REVIEW

File: [core]/es_extended/client/main.lua [hunk 1]
```diff
@@ -84,13 +84,13 @@ AddEventHandler('esx:playerLoaded', function(xPlayer, isNew, skin)
 		end)
 	end
 
-	if Config.DisableHealthRegeneration or Config.DisableWeaponWheel or Config.DisableAimAssist or Config.DisableVehicleRewards then
+	if Config.DisableHealthRegeneration then
+		SetPlayerHealthRechargeMultiplier(playerId, 0.0)
```
Comment (Gellipapa) [inline [core]/es_extended/client/main.lua:88]: I don't remember but shouldn't this be handled in a thread?

Comment (MoskalykA) [reply [core]/es_extended/client/main.lua:88]: Why ?

Comment (TheFantomas) [reply [core]/es_extended/client/main.lua:88]: @Thekuca will test that....

**Review by Thekuca — APPROVED** [review]
Everything works. Nice PR

**Pull request merged**